packages/@ember/-internals/routing/lib/system/router.ts

```diff
@@ -1735,7 +1745,7 @@ export function triggerEvent<R extends Route>(
     }
   }
 
-  let defaultHandler = defaultActionHandlers[name];
+  let defaultHandler = (defaultActionHandlers as any)[name];
```

Comment:
Can we instead requiring that `name` be `keyof typeof defaultActionHandlers` or otherwise tighten this down to avoid the `as any`? Not the end of the world if not; we can do future passes looking for `as any`s, but figured I'd check.